examples/in_game_console.js

```diff
@@ -0,0 +1,107 @@
+// ==UserScript==
+// @name         Entity Debug Tool
+// @description  https://github.com/Cazka/diepAPI
+// @version      0.0.1
+// @author       nopeless
+// @match        https://diep.io/*
+// @icon         https://www.google.com/s2/favicons?domain=diep.io
+// @namespace    https://github.com/nopeless
+// @grant        none
+// ==/UserScript==
+
+if (!window.diepAPI) return window.alert('Please install diepAPI to use this script');
+
+const { player, game, entityManager, Vector } = window.diepAPI;
+
+
+const id = "diep-ingame-console";
+
+`
```

Comment:
Did you forgot something here?

Reply:
yes I was working on a hotfix html file and forgot to update the userscript file. It is reflected in https://github.com/Cazka/diepAPI/pull/26/commits/1d2459a7f01b12892a27b61c9fd0b5c78efbeed2